README.md

```diff
@@ -35,6 +35,26 @@ For a quick reminder of these rules, just say `@AutoDelete help`.
 
 If you need extra help, say `@AutoDelete adminhelp ... message ...` to send a message to the support guild.
 
+## DockerFile
+
+For self hosting convience, i have included a dockerfile to build off of.
+
+In order for the docker container to work you will need mount the following items into the container
+
+/path/to/storage/config.yml:/autodelete/config.yml
+/path/to/storage/data/:/autodelete/data/
+
+To run the container locally you should run it like this
+
+```
+docker run -d -p 2202:2202/tcp \
+ --name Autodelete \
+ -v /opt/AutoDelete/config.yml:/autodelete/config.yml \
+ -v /opt/AutoDelete/data/:/autodelete/data/ \
+ --restart=always \
+ jacoknapp/autodelete-discord:latest
```

Comment:
I'd like to avoid naming a Dockerhub image someone else controls here. The Dockerfile itself is enough to build it, right?

Reply:
Switched this up to still have the example but not call my image directly. i did mention that images exist on dockerhub, but they are not officially supported by you. and also added build instructions for people that aren't super savvy in docker.